_sass/buttons.scss

```diff
@@ -13,6 +13,23 @@
   }
 }
 
+@keyframes grow {
+  50%{ transform: scaleX(20);}
+  50%{ transform: scaleY(20);}
+  100%{ transform: scaleX(1);}
+  100%{ transform: scaleY(1);}
+}
+
+button {
+  transform: rotate(24deg);
```

Comment:
Maybe don't do this, it's not super useable.

Reply:
Nice idea but a bit extreme. Animations should be subtle and natural

Reply:
but we want something fun though

Reply:
I can see why you would think that @hannahhch , but, I think the benefits of this outweigh the cons.

Reply:
show me the user testing!